backend/src/grouter/rdp.c

```diff
@@ -326,10 +329,17 @@ void rdp_gobackn_recv_callback (
     if(is_ack_packet) {
     	//If it is an ack packet and for the sequence number we are waiting for then we update the context
     	if(seq_num == seq_num_expected){
```

Comment:
not sure if we should still be using this as we can expect any of the sequence number we've sent and that are in the window, not just a particular one